dove/tests/test_cmd_dove_tx.rs

```diff
@@ -42,9 +42,9 @@ fn test_cmd_dove_tx_without_arguments() {
 }
 
 /// $ dove tx -n sdemo_4 -a 16 -t u8
-/// $ dove tx 'sdemo_4()' -a 16 -t u8
-/// $ dove tx 'sdemo_4(16) -t u8'
-/// $ dove tx 'sdemo_4<u8>(16)'
+/// $ dove tx sdemo_4() -a 16 -t u8
+/// $ dove tx sdemo_4(16) -t u8
+/// $ dove tx sdemo_4<u8>(16)
```

Comment:
@vladimirovmm Could you add test for multiple parameters of the script? I'm not sure it's going to work without quotes